tests/App.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { mount, render, shallow } from 'enzyme';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+
+import App from '../src/App';
```

Comment:
Make sure your code is up to date with the develop branch. You shouldn't have been able to commit without the extensions on your imports.